definitions/v7/gimmepeers.yml

```diff
@@ -96,8 +96,8 @@ search:
   inputs:
     $raw: "{{ range .Categories }}c{{.}}=1&{{end}}"
     search: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}{{ if .Query.Genre }} genre:{{ .Query.Genre }}{{ else }}{{ end }}"
-    # 0 title, 1 nfo, 2 filelist, 3 title+nfo
-    blah: "{{ if or .Query.IMDBID .Query.Genre }}3{{ else }}0{{ end }}"
+    # 0 title, 1 nfo, 2 filelist, 3 title+nfo, 4 IMDB (tt#)
```

Comment:
Is this imdbid only or also for genre?
If it's not for both then the condition will need to be changed

Reply:
Good point! I overlooked the Genre. This change is only for the IMDb id. I haven't had to dig into these indexers before, but I requested GP to make this change for IMDb, so I figured I could take the extra step to request the indexer to be updated.

Would this line do the trick?

blah: "{{ if .Query.IMDBID }}4{{ else }}0{{ end }}{{ if .Query.Genre }}3{{ else }}0{{ end }}"